### ISSUE TITLE
The requested module 'nanostores' does not provide an export named 'actionId' (v0.10.0)

### ISSUE THREAD

**Issue by felix-berlin** [nanostores/logger#18]
I migrated to nanostores v0.10.0 and removed all the deprecated `actions`.

After doing this, I still get an error from nanostores logger.

```
18:58:00 [ERROR] The requested module 'nanostores' does not provide an export named 'actionId'
  Stack trace:
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:131:21)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async ssrImport (file:///home/felix/workspace/berliner-schnauze/node_modules/.pnpm/vite@5.1.3_@types+node@20.11.20_sass@1.71.1/node_modules/vite/dist/node/chunks/dep-stQc5rCc.js:54634:24)
    at async instantiateModule (file:///home/felix/workspace/berliner-schnauze/node_modules/.pnpm/vite@5.1.3_@types+node@20.11.20_sass@1.71.1/node_modules/vite/dist/node/chunks/dep-stQc5rCc.js:54696:9)
```

[My project](https://github.com/felix-berlin/berliner-schnauze) uses Astro, Vue and nanostores with vue dev tools. This is my config:

```ts
import type { App } from "vue";
import FloatingVue from "floating-vue";
import { devtools } from "@nanostores/vue/devtools";
// import VueVirtualScroller from "vue-virtual-scroller";

import {
  $wordOfTheDay,
  $isDarkMode,
  $wordSearch,
  $filteredWordList,
  searchLength,
  $installPrompt,
  $showInstallButton,
  $showWordListFilterFlyout,
  $toastNotify,
  $activeFilterCount,
} from "../stores/index";

export default (app: App) => {
  app.use(FloatingVue, {
    themes: {
      ...FloatingVue.options.themes,
      "word-options": {
        $extend: "dropdown",
        triggers: ["click"],
        autoHide: true,
        placement: "bottom",
      },
    },
  });
  app.use(devtools, {
    $wordOfTheDay,
    $isDarkMode,
    $wordSearch,
    $filteredWordList,
    searchLength,
    $installPrompt,
    $showInstallButton,
    $showWordListFilterFlyout,
    $toastNotify,
    $activeFilterCount,
  });
  // app.use(VueVirtualScroller);
};
```

**After disabling the nanostores vue devtool part, I get no more errors.**

**Comment by oliverne** [comment]
Same on me 😥

**Comment by incraigulous** [comment]
Same with me!

**Comment by euaaaio** [comment]
`@nanostores/logger` will be updated this week with Actions API inside.
Sorry for the delay 🫶🏻

**Comment by lschierer** [comment]
I see the branch for solving this in the repository with the last commits from two weeks ago.  Is something blocking you from merging it?

**Comment by euaaaio** [comment]
Just released v0.3.0 with a fix

**Comment by lschierer** [comment]
I'm able to successfully upgrade now, thanks!!
